app/views/profile/show.html.erb

```diff
@@ -75,6 +75,20 @@
       </div>
     </div>
   </div>
+  <div class="govuk-accordion__section ">
+    <div class="govuk-accordion__section-header">
+      <h2 class="govuk-accordion__section-heading">
+        <span class="govuk-accordion__section-button" id="accordion-default-heading-2">
```

Comment:
can we call the IDs something more meaningful instead of `accordion-default-heading-2`

Reply:
Updated IDs across the page to something more meaningful